e2e/utils/delete-database.ts

```diff
@@ -1,20 +1,22 @@
 import { expect } from '@playwright/test'
 
-const defaultDbName = 'empty-db.sql.gz'
-
-export const deleteTestDatabase = async (page, dbName = defaultDbName) => {
-    const dbFileName = `${dbName}`
+export const deleteTestDatabase = async (page) => {
+    await page.locator('body').click()
 
     await page.getByRole('link', { name: 'Databases' }).click()
     await expect(page.getByRole('button', { name: 'Upload database' })).toBeVisible()
 
-    await expect(page.getByRole('cell', { name: dbFileName })).toBeVisible()
-    const testDatabaseRow = page.getByRole('row', { name: dbFileName })
-    await testDatabaseRow.getByTestId('dhis2-uicore-button').click()
+    const firstRowButton = page
+        .locator('table[data-test="dhis2-uicore-datatable"] tbody tr:first-child td[data-test="dhis2-uicore-datatablecell"]:last-child button[data-test="dhis2-uicore-button"]')
+        .first()
+
+    await firstRowButton.click()
     await page.getByRole('menuitem', { name: 'Delete' }).click()
 
-    await expect(page.getByTestId('dhis2-uicore-modalcontent')).toContainText(dbFileName)
+    await expect(page.getByTestId('dhis2-uicore-modalcontent')).toBeVisible()
     await page.getByRole('button', { name: 'Confirm' }).click()
 
-    await expect(page.getByTestId('dhis2-uicore-alertbar').getByText(new RegExp(`Successfully deleted [^/]+/${dbFileName}`))).toBeVisible()
+    const alertBar = page.locator('div[data-test="dhis2-uicore-alertbar"]')
```

Comment:
Doesn't the `alertbar` have the same locator when it shows an error? I think expecting it by it's success message makes more sense.